example/rails-51/app/views/resque.md

```diff
@@ -0,0 +1,25 @@
+# Bugsnag Rescue demo
+
+This route demonstrates how to use Bugsnag with Rescue.
+
+While testing the examples open [your dashboard](https://app.bugsnag.com) in order to see the example errors and exceptions being received.
+
+Make sure you have a Redis instance running that your test application can connect to before running these examples.
```

Comment:
Is setting this up common knowledge for most ruby developers?

Reply:
Is the issue too much information or not enough?